packages/cluster-ui/src/multiSelectCheckbox/multiSelectCheckbox.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import Select, { components } from "react-select";
+import styles from "./multiSelectCheckbox.module.scss";
+import classNames from "classnames/bind";
+
+const cx = classNames.bind(styles);
+
+const defaultSelectProps = {
+  searchable: false,
+  clearable: false,
+};
+
+export interface SelectOption {
+  label: string;
+  value: string;
+  isSelected: boolean;
+}
+
+export interface MultiSelectCheckboxProps {
+  field: string;
+  options: SelectOption[];
+  parent: any;
+  placeholder: string;
+  value?: SelectOption[];
+}
+
+const CheckboxOption = (props: any) => {
```

Comment:
I don't love how much we are using `any` here for these internal props of `react-select`. It's frustrating, because under the hood, they also use any. Could you perhaps add some comments about what these types represent if we have their type as `any`?

Reply:
I added some comment, let me know if it's enough